packages/core/sdk/__tests__/internal/net/StitchRequestClientUnitTests.ts

```diff
@@ -26,12 +26,26 @@ import { StitchRequest } from "../../../src/internal/net/StitchRequest";
 import StitchError from "../../../src/StitchError";
 import { StitchServiceErrorCode } from "../../../src/StitchServiceErrorCode";
 import StitchServiceError from "../../../src/StitchServiceError";
+import { promises } from "fs";
+/** @hidden */
+// TODO: mockito does not yet support mocking interfaces but it will soon
```

Comment:
[nit] I don't think this needs to be a TODO in the code. Get rid of the `TODO: ` but keep the rest of the comment. You can file a ticket, but I don't think it's necessary because this is just test code.